h/migrations/versions/ff02f2c58_add_the_organization_table.py

```diff
@@ -0,0 +1,28 @@
+"""
+Add the organization table.
+
+Revision ID: ff02f2c58
+Revises: 503ab9a77
+Create Date: 2018-02-08 11:00:50.420618
+"""
+
+from __future__ import unicode_literals
+
+from alembic import op
+import sqlalchemy as sa
+
+revision = 'ff02f2c58'
+down_revision = '503ab9a77'
```

Comment:
Where does this `down_revision` come from? It should match the ID of the previous revision in the logical ordering of schema changes? In other words, the `(revision, down_revision)` pairs across all the revision files should form a chain with no branches.